legate/core/store.py

```diff
@@ -1197,7 +1197,7 @@ def compute_key_partition(self, restrictions):
                 partition = Tiling(self._runtime, tile_shape, launch_shape)
             return partition
 
-    def compute_projection(self, proj_fn=None, launch_ndim=None):
+    def compute_projection(self, proj_fn=None, launch_ndim=None) -> Any:
```

Comment:
I'm pretty sure this returns integers.

Reply:
https://github.com/nv-legate/legate.core/pull/196/commits/6a5897c32657ab74b13d989d99e01b2a577a5107